authui/src/authflowv2/base.css

```diff
@@ -32,4 +39,20 @@
     --c-grey-600: #303135;
     --c-grey-700: #131315;
   }
+
+  /* Border Radius */
+  :root {
+    --border-radius-full: 9999px;
+    --border-radius-sqaure: 0px;
+    --border-radius-rounded-square: 14px;
```

Comment:
According to our offline discussion, we do not need these 3 variables. The border radius of primary button should be derived from its font-size.

Reply:
@louischan-oursky I think we still need these 3 variables. Only `rounded-square` should be calculated from the font-size.